Add support for additional flags (#53)
cisagov/ansible-role-clamav: README.md

| Variable | Description | Default | Required |
|----------|-------------|---------|----------|
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |

> | clamav_scan_copy | Whether to copy infected files to quarantine folder. | false | No |

Reviewer: ```suggestion
| clamav_scan_copy | Whether to copy infected files to quarantine folder. | `false` | No |
```

> | clamav_scan_exclude_directories | Directory trees to exclude from scan operations. | `{ /dev, /proc, /sys, /var/spool/clamav }` | No |

Reviewer 2: This is `list`, not a `dict`:
```suggestion
| clamav_scan_exclude_directories | A list of regexes matching irectory trees that are to be excluded from scan operations. | `[^/dev, ^/proc, ^/sys, ^/var/spool/clamav]` | No |
```

I also updated the default values to match those in `defaults/main.yml` and tweaked the variable description to mention that the list entries are regexes.

| clamav_scan_extra_flags | Additional flags to pass to clamscan (see clamscan man page for reference).  | `[]` | No |

> | clamav_scan_move | Whether to move infected files to a quarantine directory. | false | No |

Reviewer: ```suggestion
| clamav_scan_move | Whether to move infected files to a quarantine directory. | `false` | No |
```

> | clamav_scan_quarantine_dir | Directory to store infected files. | `/var/spool/clamav` | No |

Reviewer: Minor thing but I prefer to avoid abbreviations when feasible. I believe I've appropriately suggested all other appearances of this variable but apologies if I've missed one.

```suggestion
| clamav_scan_quarantine_directory | Directory to store infected files. | `/var/spool/clamav` | No |
```

> | clamav_scan_exclude_directories | directory trees to exclude from scan operations         | { /dev, /sys, /proc } | No |
> | clamav_scan_move                | wether to move infected files to quarantine folder      | false | No |
> | clamav_scan_copy                | wether to copy infected files to "quarantine" folder    | false | No |
> | clamav_scan_quarantine_dir      | where to store infected files                           | '/var/spool/clamav' | No |
> | clamav_scan_quarantine_owner    | permissions on quarantine directory                     | 'root' | No |
> | clamav_scan_quarantine_group    | permissions on quarantine directory                     | 'root' | No |
> | clamav_scan_quarantine_mode     | permissions on quarantine directory                     | '0750' | No |
> | clamav_scan_extra_flags         | additional flags (see clamscan man page for reference)  | [] | No |

Reviewer 3: Alphabetize, clean up wording, and fix typos/formatting:
```suggestion
| clamav_scan_copy | Whether to copy infected files to quarantine folder. | false | No |
| clamav_scan_exclude_directories | Directory trees to exclude from scan operations. | `{ /dev, /proc, /sys, /var/spool/clamav }` | No |
| clamav_scan_extra_flags | Additional flags to pass to clamscan (see clamscan man page for reference).  | `[]` | No |
| clamav_scan_move | Whether to move infected files to a quarantine directory. | false | No |
| clamav_scan_quarantine_dir | Directory to store infected files. | `/var/spool/clamav` | No |
| clamav_scan_quarantine_group | Group owner to apply to quarantine directory. | `root` | No |
| clamav_scan_quarantine_mode | Permissions to apply to quarantine directory. | `0750` | No |
| clamav_scan_quarantine_owner | Owner to apply to quarantine directory. | `root` | No |
```

| clamav_configuration_backup | Whether or not to backup configuration files before changing. | `false` | No |
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
| clamav_freshclam_configuration | A dictionary of values to set in the freshclam configuration file. | `{}` | No |
| clamav_scan_copy | Whether to copy infected files to quarantine folder. | `false` | No |

> | clamav_scan_exclude_directories | A list of regexes matching irectory trees that are to be excluded from scan operations. | `[^/dev, ^/proc, ^/sys, ^/var/spool/clamav]` | No |

Reviewer: ```suggestion
| clamav_scan_exclude_directories | A list of regexes matching directory trees that are to be excluded from scan operations. | `[^/dev, ^/proc, ^/sys, ^/var/spool/clamav]` | No |
```
